### DISCUSSION TITLE
Some Questiion for new contrubutors in joomla

### DISCUSSION THREAD

**Discussion by Kaushik1216** [joomla/joomla-cms#39439] [Q&A] [answered]
Hi I am Kaushik Vishwakarma.I want to contribute in Joomla
So I have some question about it .for examples

1.Is there is any IRC ,Glip or other channels where all contributors chat /discuss ? I found google groups but that are Inactive from long days.

2.In which repo should I start contributions Joomla-cms or Joomla-projects or both?

I am Glad if you answer my question
Looking forward to work with you…

**Comment by bembelimen** [comment, marked as answer]
Join [joom.la/chat](https://joom.la/chat) for discussing etc. Contributing happens here in this repo.

**Comment by Arunjagan12** [comment]
Greetings sir,I am Arun studying second year in cse.I am interested to contribute in upcoming gsoc 2023 with your organisation as your project sparked my interest.I am familiar with html css and javascript and worked as a web dev intern as well.Can u help me in solving the issues of your project(schema org) as it's slightly difficult for me to understand.Can u also provide some tips to crack gsoc 2023 as well??

**Reply by N6REJ** [reply]
you should ask in your own question not hijack an already answered question
